XRTK-Core/Assets/XRTK.Seed/MixedRealityPackageSeed.cs

```diff
@@ -66,7 +66,7 @@ private static void Run()
 
                         File.WriteAllText(manifestFilePath, text);
 
-                        Client.Add("com.xrtk.core");
+                        Client.Add("com.xrtk.sdk");
```

Comment:
Assuming core gets added as well since SDK has a dependancy on it?

Reply:
Correct